Kconfig.zephyr

```diff
@@ -575,6 +575,7 @@ endif # BUILD_OUTPUT_UF2
 
```

Comment:
@marc-hb  This change is very useful! Thanks.

Also for the incremental builds test, wouldn't make more sense to touch a header file like  `./include/zephyr/kernel.h` for example.